### PR TITLE
docs: add Changelog for Ireland release

### LINKED ISSUE #118: Add Changelog for Ireland release
Add Changelog for Ireland release (V2)
The Changelog would be based on the changelog-generator
https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/cd-management/job/changelog-generator/

### PR DESCRIPTION
**Pull request by cloudxxx8**
Signed-off-by: Cloud Tsai <cloudxxx8@gmail.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-camera-go/blob/master/.github/CONTRIBUTING.md

## Issue Number:
Fix #118

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Style
+- Use pascal case in the sample Profiles ([#b9d8927](https://github.com/edgexfoundry/device-camera-go/commits/b9d8927))
+### Bug Fixes 🐛
+- address Lenny's feedback about authMode none to skip getCredentials - skip GetCredentials from secret provider if AuthMethod is none ([#c058d96](https://github.com/edgexfoundry/device-camera-go/commits/c058d96))
+- Remove retry items of SecretStore config and update secret path - go-mod-bootstrap has implemented the addition of prefix /v1/secret/edgex/ for the Path property of SecretStore config section, so we just use the service specific secret path in Toml files - removed retry related items in "SecretStore" config section ([#d51bbe5](https://github.com/edgexfoundry/device-camera-go/commits/d51bbe5))
```
Comment (lenny-intel) [inline CHANGELOG.md:16]: Remove as these are dev churn during dev cycle

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- **build:** update go.mod to go 1.16 ([#d43abaa](https://github.com/edgexfoundry/device-camera-go/commits/d43abaa))
+- **build:** update Dockerfiles to use go 1.16 ([#ba22a61](https://github.com/edgexfoundry/device-camera-go/commits/ba22a61))
+- **snap:** '-go' suffix removed from device name ([#2a3a7a1](https://github.com/edgexfoundry/device-camera-go/commits/2a3a7a1))
+- **snap:** run 'go mod tidy' ([#c9606bd](https://github.com/edgexfoundry/device-camera-go/commits/c9606bd))
+- **snap:** update go to 1.16 ([#5bc5291](https://github.com/edgexfoundry/device-camera-go/commits/5bc5291))
+- **snap:** update snap v2 support ([#5a3c97e](https://github.com/edgexfoundry/device-camera-go/commits/5a3c97e))
```
Comment (lenny-intel) [inline CHANGELOG.md:22]: Should be under Build section

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+### Code Refactoring ♻
+- bump dependency version and update import path ([#9562c59](https://github.com/edgexfoundry/device-camera-go/commits/9562c59))
```
Comment (lenny-intel) [inline CHANGELOG.md:24]: Remove as it is dev churn

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- remove unimplemented InitCmd/RemoveCmd configuration ([#d1479f8](https://github.com/edgexfoundry/device-camera-go/commits/d1479f8))
+- Remove the dependency of pkg/errors ([#1083b87](https://github.com/edgexfoundry/device-camera-go/commits/1083b87))
+- Change PublishTopicPrefix value to be 'edgex/events/device' ([#0734506](https://github.com/edgexfoundry/device-camera-go/commits/0734506))
```
Comment (lenny-intel) [inline CHANGELOG.md:27]: Remove as it is dev churn

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Comment by lenny-intel** [comment]
Please add `Change Logs for EdgeX Dependencies` section like on ASC and App SDK change log
https://github.com/edgexfoundry/app-service-configurable/blob/master/CHANGELOG.md
https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/CHANGELOG.md

Please make all all these same changes (above and in comments) to the other change logs. THX!

**Comment by cloudxxx8** [comment]
@lenny-intel 
In the app service CHANGELOGS
https://github.com/edgexfoundry/app-service-configurable/blob/master/CHANGELOG.md
https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/CHANGELOG.md

We should use v2.0.0 tags instead of the master tags because `master` branch will be renamed to `main` branch
Also, the URL of go-mod-configuration is wrong

- [go-mod-bootstrap](https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/CHANGELOG.md)
- [go-mod-core-contracts](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/CHANGELOG.md)
- [go-mod-messaging](https://github.com/edgexfoundry/go-mod-messaging/blob/master/CHANGELOG.md)
- [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry/blob/master/CHANGELOG.md) 
- [go-mod-configuration](https://github.com/edgexfoundry/go-mod-configuration) (indirect dependency)
- [go-mod-secrets](https://github.com/edgexfoundry/go-mod-secrets/blob/master/CHANGELOG.md) (indirect dependency)

should become

- [go-mod-bootstrap](https://github.com/edgexfoundry/go-mod-bootstrap/blob/v2.0.0/CHANGELOG.md)
- [go-mod-core-contracts](https://github.com/edgexfoundry/go-mod-core-contracts/blob/v2.0.0/CHANGELOG.md)
- [go-mod-messaging](https://github.com/edgexfoundry/go-mod-messaging/blob/v2.0.1/CHANGELOG.md)
- [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry/blob/v2.0.0/CHANGELOG.md)
- [go-mod-configuration](https://github.com/edgexfoundry/go-mod-configuration/blob/v2.0.0/CHANGELOG.md) (indirect dependency)
- [go-mod-secrets](https://github.com/edgexfoundry/go-mod-secrets/blob/v2.0.0/CHANGELOG.md) (indirect dependency)

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**